src/components/NewLocationPage/ChartTabs/ChartTab.tsx

```diff
@@ -14,7 +14,8 @@ const ChartTab: React.FC<{
   metricName: string;
   subLabel?: string[];
   metricValueInfo: ValueInfo;
-}> = ({ metricName, subLabel, metricValueInfo }) => {
+  showColorIndicator?: boolean;
+}> = ({ metricName, subLabel, metricValueInfo, showColorIndicator = true }) => {
```

Comment:
Added `showColorIndicator` as an optional prop to the `ChartTab` component here instead of implementing conditional logic in `getValueInfo()` in `Groupings.tsx`, since this will give us more flexibility if we want to display (or not display) a color indicator in any chart tab in the future.